assets/hmbkp.js

```diff
@@ -139,21 +139,6 @@ var BackUpWordPressAdmin = (function($){
 			);
 		} );
 
-		jQuery( document ).one( 'click', '.hmbkp_send_error_via_email', function ( e ) {
```

Comment:
I think this was intentionally removed as unused code, I can't find a reference to either the class `hmbkp_send_error_via_email` or the function `hmbkp_email_error` in the rest of the codebase.